packages/cloudflare/src/api/overrides/incremental-cache/kv-incremental-cache.ts

```diff
@@ -23,69 +21,30 @@ class KVIncrementalCache implements IncrementalCache {
     key: string,
     isFetch?: IsFetch
   ): Promise<WithLastModified<CacheValue<IsFetch>> | null> {
-    const cfEnv = getCloudflareContext().env;
-    const kv = cfEnv.NEXT_INC_CACHE_KV;
-    const assets = cfEnv.ASSETS;
-
-    if (!(kv || assets)) {
-      throw new IgnorableError(`No KVNamespace nor Fetcher`);
-    }
+    const kv = getCloudflareContext().env.NEXT_INC_CACHE_KV;
```

Comment:
We can do `getCloudflareContext().env.NEXT_INC_CACHE_KV` in the ctor (see #538)

Reply:
Sorry i didn't know you edited the comment directly.

Reply:
We still cannot do this.

<img width="815" alt="image" src="https://github.com/user-attachments/assets/d95f0884-ce21-4cab-8a54-39e4643bb89c" />

Reply:
> Sorry i didn't know you edited the comment directly.

I should have added a comment instead, sorry fo that.

When/how is this error triggered?
This class should not be used while `next dev`ing

Reply:
It comes from getCloudflareContext -> getCloudflareContextSync. It throws an error because there was no cloudflare context initialised on the global scope

Reply:
We need to dig into that, it should not happen as it should be initialized at top level in worker.ts.

Also we should drop `next dev` only code from the Cloudflare build.